README.md

```diff
@@ -35,6 +35,20 @@ Edgehog Device Runtime is a reference implementation of
 Astarte interfaces describe how data are exchanged with the remote instance, and what kind of
 features are implemented.
 
+## Configuration file
+Create a TOML configuration file either in /etc/edgehog.conf or ./edgehog.conf
```

Comment:
`Edgehog Device Runtime can be configured using a [TOML](https://en.wikipedia.org/wiki/TOML) located either in $PWD/edgehog.conf or  /etc/edgehog.conf.`